akka-actor-tests/src/test/scala/akka/actor/ReceiveTimeoutSpec.scala

```diff
@@ -113,5 +113,25 @@ class ReceiveTimeoutSpec extends AkkaSpec {
       ticks.cancel()
       system.stop(timeoutActor)
     }
+
+    "get timeout while receiving NotInfluenceReceiveTimeout messages scheduled with Timers" taggedAs TimingTest in {
+      val timeoutLatch = TestLatch()
+
+      class ActorWithTimer() extends Actor with Timers {
+        timers.startPeriodicTimer("transparenttick", TransperentTick, 100.millis)
+        timers.startPeriodicTimer("identifyTick", Identify(None), 100.millis)
+
+        context.setReceiveTimeout(1 second)
+        def receive: Receive = {
+          case ReceiveTimeout  ⇒ timeoutLatch.open
+          case TransperentTick ⇒
```

Comment:
the test should also verify that this is received, I think that doesn't work now

Reply:
Yeah, confirmed that it doesn't work as expected right now. Added a counter to validate.